haystack/agents/base.py

```diff
@@ -208,7 +219,7 @@ def _run_tool(
     ) -> Union[Tuple[Dict[str, Any], str], Dict[str, Any]]:
         if not self.has_tool(tool_name):
             raise AgentError(
-                f"Cannot use the tool {tool_name} because it is not in the list of added tools {self.tools.keys()}."
+                f"The {tool_name} wasn't added to the Agent tools: {self.tools.keys()}."
```

Comment:
`The {tool_name}`-> `The tool {tool_name}` otherwise it's harder to understand that `{tool_name}` is a tool.